### PR TITLE
Cranskhaft with aasdk next

### PR DESCRIPTION
**Pull request by dicetrash**
Make opencardev/openauto work with opencardev/aasdk newdev

### CODE REVIEW

File: src/autoapp/Service/AndroidAutoEntity.cpp [hunk 1]
```diff
@@ -238,6 +238,20 @@ void AndroidAutoEntity::onShutdownRequest(const aasdk::proto::messages::Shutdown
...
+    OPENAUTO_LOG(info) << "[AndroidAutoEntity] ping request ";
+    
+    auto promise = aasdk::channel::SendPromise::defer(strand_);
+    promise->then([]() {}, std::bind(&AndroidAutoEntity::onChannelError, this->shared_from_this(), std::placeholders::_1));
+    controlServiceChannel_->sendPingRequest(request, std::move(promise));
+}
+
+void AndroidAutoEntity::onVoiceSessionRequest(const aasdk::proto::messages::VoiceSessionRequest& request)
+{
+	OPENAUTO_LOG(error) << "[AndroidAutoEntity] voice session request not implemented";
```
Comment (dicetrash) [inline src/autoapp/Service/AndroidAutoEntity.cpp:252]: Note - I couldn't find anything that references "voice session" only video session and has more then just start and stop

**Comment by matt2005** [comment]
Does this pr compile for you? I've tried to compile it, but am getting the following error.
``` 

/home/pi/workspace/buildsystem/openauto/src/btservice/AndroidBluetoothServer.cpp:27:10: fatal error: aasdk_proto/WifiInfoRequestMessage.pb.h: No such file or directory
   27 | #include <aasdk_proto/WifiInfoRequestMessage.pb.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/btservice.dir/build.make:121: CMakeFiles/btservice.dir/src/btservice/AndroidBluetoothServer.cpp.o] Error 1
```

**Comment by dicetrash** [comment]
My aasdk_next was compiled with WifiInfoRequestMessage.proto still intact so yes it compiled, I can add an MR to add the three deleted proto files back into aasdk, not familiar enough with the project to know why they where deleted.

**Comment by matt2005** [comment]
I'll verify it builds and test it over the weekend

**Comment by Fabio3rs** [comment]
I checkout this pull request, and I needed to make the following edit to be able to compile:

```
diff --git a/src/autoapp/Projection/OMXVideoOutput.cpp b/src/autoapp/Projection/OMXVideoOutput.cpp
index f8f3808..6a46380 100644
--- a/src/autoapp/Projection/OMXVideoOutput.cpp
+++ b/src/autoapp/Projection/OMXVideoOutput.cpp
@@ -23,7 +23,7 @@ extern "C"
 #include <bcm_host.h>
 }
 
-#include <f1x/aasdk/Common/Data.hpp>
+#include <aasdk/Common/Data.hpp>
 #include <f1x/openauto/autoapp/Projection/OMXVideoOutput.hpp>
 #include <f1x/openauto/Common/Log.hpp>
 
```

File: src/autoapp/Service/SensorService.cpp
Comment (Chaos02) [inline src/autoapp/Service/SensorService.cpp:1]: it seems you have tried to use a different libgps version than what was previously used. is this intentional?
You seem to have resolved the resulting problems but someone really should write down the versions used for compilation (or at least the distro version!)
From what I can tell the original working version is `libgps-dev=3.17-5`

**Pull request merged**